config/longrun_configs/longrun_aquaplanet_allsky_tvinsol_0M_slabocean.yml

```diff
@@ -1,3 +1,9 @@
-atmos_config_file: "config/longrun_configs/longrun_aquaplanet_allsky_tvinsol_0M_slabocean.yml"
+albedo_model: "RegressionFunctionAlbedo"
+approximate_linear_solve_iters: 2
+atmos_config_file: "config/longrun_configs/amip_target_edonly.yml"
```

Comment:
If the intention here is to retain one aquaplanet longrun (based on the filename) we could revert the `atmos_config_file` to the original slabocean version.

Reply:
In the coupler, running aquaplanet is controlled by specifying `mode_name: "slabplanet_aqua"` (which I should have added here and will update). Is there a problem with using the "AMIP" atmos config, but setting up aquaplanet in the coupler via this config option?

Reply:
Ah it uses `toml: [toml/longrun_aquaplanet.toml]` in the atmos model; I think in this case with `mode_name: slabplanet_aqua` we can leave it as is i.e. `atmos_config_file = config/longrun_configs/longrun_aquaplanet_allsky_tvinsol_0M_slabocean.yml` (both use ED-only, so `decaywithheightdiffusion` is the only other viable option which has a clear identifier elsewhere)